contrib/ansible/roles/elasticsearch/tasks/main0.yml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
I dont't understand the purpose of this main0 file, something commited by error ?

Reply:
Committed by error :-)